### DISCUSSION TITLE
How to use loadFont()?

### DISCUSSION THREAD

**Discussion by jaydar10** [replit/kaboom#589] [Q&A] [answered]
Okay, so I'm making a Mario platformer template on Replit (obviously, using Kaboom.js), and when I made a game over screen, I found that I just didn't like the default font in Kaboom.
<br>So I looked for a font, finding that I liked `"Press Start 2P"`, however I have no idea what the `src` parameter on the `loadFont()` function wants from me.
<br>I know it wants a `.png`, `.jpg`, or picture file of some sort, but what should _in **that file**_?

**Comment by jaydar10** [comment, marked as answer]
Okay, after looking at some more Kaboom games on Replit, I figured it out. The picture Kaboom is looking for is a screenshot of the `.ttf` file, and then, from what I know, you have to name it the same as the `.ttf` file itself, ex; if the `.ttf` file was called `StupidFont-Norma.ttf`, you'd name it `StupidFont-Normal.png`.

**Reply by lajbel** [reply]
Hey hi!

You are probably using the latest version of Kaboom, so the fonts work in png, you can perfectly load fonts that the width and length of the letters are the same in each character, something like a spritesheet.

Example: 
![image](https://user-images.githubusercontent.com/71136486/170847018-3696b6f3-53dd-4c23-894e-536cdce2fa25.png)

```js
loadFont("unscii", "/fonts/unscii_8x8.png", 8, 8); // each char has 8 of w and h
```

It is currently possible to use TrueType fonts in the latest `next` version of Kaboom. Check [this](https://replit.com/@slmjkdbtl/KaboomTrueType) example

On the other hand, you can use [font2bitmap](https://stmn.itch.io/font2bitmap) to convert a Google Fonts font to BitMap .png

**Reply by jaydar10** [reply]
Okay, I think my project is kind of just... cursed?
I swear my synax is correct, as I put in:
```
loadFont("pressStart", "/fonts/PressStart2P-Regular.ttf", 8, 8);
	const coinsLabel = add([
		text(coins, { font: "pressStart", size: 24, } ),
		pos(24, 24),
		fixed(),
	])
```
And I'd use the same syntax on the example, and that'd work. It's strange...
